File: src/test/java/day4/ScratchcardsTest.java
package day4;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class ScratchcardsTest {

    @Test
    void firstStar() {
        Scratchcards scratchcards = new Scratchcards();
        assertEquals(13, scratchcards.firstStar());
    }

    @Test
    void secondStar() {
        Scratchcards scratchcards = new Scratchcards();
        assertEquals(30, scratchcards.secondStar());
    }
}

File: src/test/kotlin/day04/ScratchcardsTest.kt
package day04

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

internal class ScratchcardsTest {
    @Test
    fun firstStar() {
        assertEquals(13, Scratchcards().firstStar())
    }

    @Test
    fun secondStar() {
        assertEquals(30, Scratchcards().secondStar())
    }
}